lib/main.dart

```diff
@@ -18,11 +18,9 @@ class ECellApp extends StatelessWidget {
   final SharedPreferences sharedPreferences = sl.get<SharedPreferences>();
   @override
   Widget build(BuildContext context) {
-    // ignore: unused_local_variable
-    String token = sharedPreferences.getString(S.tokenKey);
+    String token = sl.get<SharedPreferences>().getString(S.tokenKey);
```

Comment:
this line is throwing an error while analysis. remove it (and the above line 18) for now. The todo that is below is sufficient to remind to add the variable.